.github/workflows/Linux.yml

```diff
@@ -0,0 +1,102 @@
+name: Linux CI
+
+on: [push, pull_request]
+
+defaults:
+  run:
+    shell: bash
+    working-directory: Code
+
+jobs:
+  linux-ci:
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - cfg: Build & Tests
+            gcc: 10
+            clang: 12
+            os: ubuntu-20.04
+          - cfg: Build
+            gcc: 7
+            clang: 9
+            os: ubuntu-18.04
+          - cfg: ASan
+            gcc: 10
+            clang: 12
+            os: ubuntu-20.04
+          - cfg: MSan
+            gcc: 10
+            clang: 12
+            os: ubuntu-20.04
+            can-fail: true
+          - cfg: TSan
+            gcc: 10
+            clang: 12
+            os: ubuntu-20.04
+            can-fail: true
+
+    name: Linux (${{ matrix.cfg }}, GCC ${{ matrix.gcc }}, Clang ${{ matrix.clang }})
+    runs-on: ${{ matrix.os }}
+    continue-on-error: ${{ matrix.can-fail || false}}
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - uses: ./.github/actions/DownloadFBuild
+        with:
+          version: 1.03
```

Comment:
Should this use the current version? (v1.05)?

Would it be better to keep the version inside the DownloadFBuild action so that it can be updated in only one place when new versions are released?

Reply:
> Should this use the current version? (v1.05)?

I just copied the version from `.travis.yml`. Will try with 1.05.

> Would it be better to keep the version inside the DownloadFBuild action so that it can be updated in only one place when new versions are released?

I agree. It seems I was too busy playing with syntax. Added a default value to `version` in `DownloadFBuild/action.yml` and removed `with` sections in workflow files.